airbyte-integrations/connectors/source-github/source_github/streams.py

```diff
@@ -295,6 +300,9 @@ class Organizations(GithubStream):
     API docs: https://docs.github.com/en/rest/reference/orgs#get-an-organization
     """
 
+    # GitHub pagination could be from 1 to 100.
+    page_size = 100
```

Comment:
And another one "page size" here. Is it something different?

Reply:
Organizations streams work as a slightly separate branch, the `__init__` method is overridden there, and there are no large streams for Organizations, so we can safely use 100 as the page size.